src/tcpedit/tcpedit_opts.def

```diff
@@ -58,6 +58,49 @@ you can reuse the same seed value to recreate the traffic.
 EOText;
 };
 
+flag = {
+    name        = range_count_src;
+    descrip     = "Set the count of source ip address range";
+    arg-type    = number;
+    arg-range   = "1->1048575";
+    arg-default = 1;
+    max         = 1;
+    doc         = <<- EOText
+Increament source ip address (ipv4) within specified range.
```

Comment:
Typo - also, need better description of what this option (or this entire feature) does.

I may consider this for next major release, but for now I am just doing maintenance updates. Major release may be around Jan. 2023.